src/components/molecules/UiTile/UiTile.vue

```diff
@@ -2,10 +2,10 @@
   <UiButton
     :id="tileId"
     :class="[
-      'ui-button--outlined ui-tile',{ 'ui-tile--is-checked': isChecked }
+      'ui-button--outlined ui-tile',
+      { 'ui-tile--is-checked': isChecked },
     ]"
-    role="radio"
-    :aria-checked="`${isChecked}`"
+    type="submit"
```

Comment:
@sonya0504 I think about `type=submit` here. IMO is submitted in `UiSimpleQuestion` because there have context and there is a submit button.